### ISSUE TITLE
How to fine-tuning?

### ISSUE THREAD

**Issue by Zzzzy411** [deeppomf/DeepCreamPy#88]
I want to continue training this network by fine-tuning in my own datasets. How to preprocess my training  data? Adding the random mask and crop the image in size of 512*512? Thanks for your reply.

**Comment by Zzzzy411** [comment]
I use the Step3 - UNet Architecture.ipynb code in MathiasGruber / PConv-Keras for fine-tuning the trained network "model.h5".  But there is a error in model.fit(): "Error when checking model input: the list of Numpy arrays that you are passing to your model is not the size the model expected. Expected to see 3 array(s), but instead got the following list of 2 arrays: " .  I dont understand the difference between pconv_hybrid_model.py and pconv_model.py. How to solve this problem?  I really appreciate it.

**Comment by deeppomf** [comment]
Yes, random mask and crop image to 512 x 512
I used the Step 4 code.
